nucleus/pom.xml

```diff
@@ -162,7 +137,7 @@
             </snapshots>
         </repository>
 
-        <!--  
+        <!--
```

Comment:
Since we're cleaning up, can we also get rid of the commented out parts of pom?

Reply:
Yes and no, only if we create documentation how to use fake local repo for patched projects, this is somethink like howto for beginers :-)
And no for another reason - target of this issue is not cleanup, I only moved common things to a new parent to avoid new duplications.
I have to look how much conflicts I created to your "BOM" issue, but I want to minimize that.

Reply:
I will redo BOM in next sprint, so no worries about conflicts.